cmd/ttn-lw-cli/commands/contact_info.go

```diff
@@ -173,6 +175,35 @@ func contactInfoCommands(entity string, getID func(cmd *cobra.Command, args []st
 			return io.Write(os.Stdout, config.OutputFormat, updatedInfo)
 		},
 	}
+	validate := &cobra.Command{
+		Use:   "validate",
+		Short: "Validate contact info",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			reference, _ := cmd.Flags().GetString("reference")
+			if reference == "" {
+				return errNoReference
+			}
+			token, _ := cmd.Flags().GetString("token")
+			if token == "" {
+				return errNoValidationToken
+			}
```

Comment:
@htdvisser if these are always required, shouldn't they be taken from `args`?

Reply:
Already commented that in a previous review: https://github.com/TheThingsNetwork/lorawan-stack/pull/1533#discussion_r344237042